src/renderer/tvgShape.h

```diff
@@ -297,13 +297,16 @@ struct Shape::Impl : Paint::Impl
 
     Result strokeDash(const float* pattern, uint32_t cnt, float offset)
     {
-        if ((cnt == 1) || (!pattern && cnt > 0) || (pattern && cnt == 0)) {
+        if ((!pattern && cnt > 0) || (pattern && cnt == 0)) {
             return Result::InvalidArguments;
         }
 
+        auto len = 0.0f;
```

Comment:
I think this is no point. it would be better to remove the exceptional handling for 304 ~ 309 here and just let it result in a broken visual.

Reply:
So I need to add handling for such cases in each engine individually. For now, SW and GL will hang.

Reply:
@mgrudzinska If it's simple to do, then let's go for it. Adding iteration for unusual exception handling is excessive, IHO.

Reply:
@hermet Please check now. I'm not making any additional iterations, just using the existing ones.